sphinx_rtd_theme/footer.html

```diff
@@ -2,10 +2,10 @@
   {% if (theme_prev_next_buttons_location == 'bottom' or theme_prev_next_buttons_location == 'both') and (next or prev) %}
     <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
       {% if next %}
-        <a href="{{ next.link|e }}" class="btn btn-neutral float-right" title="{{ next.title|striptags|e }}" accesskey="n">{{ _('Next') }} <span class="fa fa-arrow-circle-right"></span></a>
+        <a href="{{ next.link|e }}" class="btn btn-neutral float-right" title="{{ next.title|striptags|e }}" accesskey="n" rel="next" rev="prev">{{ _('Next') }} <span class="fa fa-arrow-circle-right"></span></a>
```

Comment:
> This should no be both next and prev, no?

I think you can use both. I found an example in the W3C HTML 5.1 Recommendation, but it uses the link element. I think it can be applied to the a element the same way. Example taken from https://www.w3.org/TR/html51/document-metadata.html#the-link-element

> Document with URL "chapter2.html"
> 
> `<link href="chapter1.html" rel="prev" rev="next">`
> `<link href="chapter3.html" rel="next" rev="prev">`